css/ampshared.css

```diff
@@ -539,6 +539,26 @@ amp-accordion {
   display: block !important;
 }
 
+/**
+ * This media query is used instead of @supports due to low compatibility.
+ * At the time of this change (4/6/2021) no version of Webkit support @supports queries.
+ *
+ * These provide low-specificity UI styles to prevent CLS for the header and content 
```

Comment:
Suggestion: Similar to the 1.0 accordion CSS, let's include a note here that the same styles are applied in 0.1 via `i-amphtml-accordion-header` class. Ideally these styles and the ones there should have tests or at least mutually referring comments ensuring they stay in sync if one changes.

Reply:
> Ideally these styles and the ones there should have tests or at least mutually referring comments ensuring they stay in sync if one changes.

This is a good idea, I'll add this in a new PR once this lands.